project-gallery/src/components/Canvas.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import "../Gallery.css";
+
+import { CanvasProps } from "../types/types";
+
+const Canvas: React.FC<CanvasProps> = ({ selectedImage }) => {
+  return (
+    <main className="main">
+      <div className="artboard">
+        <section className="canvas">
+          {selectedImage && (
+            <img
+              className="canvas__img"
+              src={selectedImage}
+              alt="Selected Large Image"
+            />
+          )}
```

Comment:
use a alt in here. not to be empty place for showing image.
"Please select an image from sidebar"